product_docs/docs/pgd/5/reference/pgd-settings.mdx

```diff
@@ -96,17 +96,21 @@ This parameter can be set only by bdr_superuser or superuser roles.
 
 Possible options are:
 
--   off &mdash; Don't use global locking for DDL operations.
--   on &mdash; Use global locking for all DDL operations.
+-   all &mdash; Use global locking for all DDL operations.
 -   dml &mdash; Use global locking only for DDL operations that need to prevent
     writes by taking the global DML lock for a relation.
+-   off &mdash; Don't use global locking for DDL operations.
 
 A `LOG`-level log message is emitted to the PostgreSQL server logs whenever
 `bdr.ddl_replication` is set to `off`. Additionally, a `WARNING` message is
 written whenever any global locking steps are skipped due to this setting. It's
 normal for some statements to result in two `WARNING` messages: one for skipping
 the DML lock and one for skipping the DDL lock.
 
+For backwards compatibility, `bdr.ddl_locking` supports aliases. 
+`on` and `true` are an alias for `all`. 
+`false` is an alias for `off`. 
+
```

Comment:
Hm.. our tests are mostly using `on` and `off` still... :thinking:

Reply:
I know. :) @PJMODOS said (and I concur) that it should be all/dml/off (as on/dml/off is two states with a bonus state, and people seeing on and off won't intuit there are other states).